### PR TITLE
New Auth | Remove tracking dimensions call not to break js on auth

### PR DESCRIPTION
**Pull request by bkoval**
## Description

We cannot send a request to http:// domain from https:// served page, it triggers mixed content error and breaks javascript on the page.

## Reviewers

@vforge @rogatty @Wikia/social-frontend 

https://wikia-inc.atlassian.net/browse/SOC-2819

### CODE REVIEW

File: server/app/views/_partials/tracking-ua-auth.hbs [hunk 1]
```diff
@@ -5,44 +5,25 @@
...
-							14: data.dimensions[14],
-							15: 'No',
-							19: data.dimensions[19],
-							25: data.dimensions[25]
-						});
-					}
-				}
-			});
+			if (ua.initialize({
+						4: 'mercury',
```
Comment (vforge) [inline server/app/views/_partials/tracking-ua-auth.hbs]: Can you hardcode missing dimensions? If you'll do we'll have same dimensions sent on all our pages - that was the point of having the call in the first place.

Comment (rogatty) [inline server/app/views/_partials/tracking-ua-auth.hbs]: FYI:
- 3: wiki vertical
- 14: show ads
- 19: article type
- 25: namespace

Comment (bkoval) [inline server/app/views/_partials/tracking-ua-auth.hbs]: I don't feel like we ever needed these on auth pages. Do you feel like adding them all set to 'false' or null makes much sense?

Comment (vforge) [inline server/app/views/_partials/tracking-ua-auth.hbs]: I would still add them - even with `null/false` - just for the sake of next person who will read the code and wonder why we don't have some dimensions.

Comment (bkoval) [inline server/app/views/_partials/tracking-ua-auth.hbs]: 👍

File: server/app/views/_partials/tracking-ua-auth.hbs [hunk 2]
```diff
@@ -2,44 +2,29 @@
...
-							14: data.dimensions[14],
-							15: 'No',
-							19: data.dimensions[19],
-							25: data.dimensions[25]
-						});
-					}
-				}
-			});
+			if (ua.initialize({
+						3: null,
```
Comment (rogatty) [inline server/app/views/_partials/tracking-ua-auth.hbs]: Too much tabs, I think.

**Comment by rogatty** [comment]
Minor formatting issue but other than that LGTM

File: server/app/views/_partials/tracking-ua-auth.hbs (hunk 2 above)
Comment (bkoval) [inline server/app/views/_partials/tracking-ua-auth.hbs]: My IDE formats it this way. You sure? ;>

**Comment by vforge** [comment]
LGTM

**Comment by rogatty** [comment]
LGTM

**Pull request merged**